mat64/hogsvd_example_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mat64_test
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gonum/matrix/mat64"
+)
+
+func ExampleHOGSVD() {
+	// Perform an HOGSVD factorization on food production/consumption data for the
+	// three years 1990, 2000 and 2014.
+	//
+	// See Ponnapalli et al. doi:10.1371/journal.pone.0028072 and
+	// Alter at al. doi:10.1073/pnas.0530258100 for more details.
+	var gsvd mat64.HOGSVD
```

Comment:
Just because these two are in the same place, and will often be viewed together (to compare the two close but different routines). In other code I wouldn't care as much.